jjb/jobs/template.yaml

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Jobs definitions.
+---
+- job-template:
+    name: 'kata-containers-{repo}-{os}-{ci_job}'
+    description: |
+      Test pull requests for Kata Containers.
+        Repository - {repo}
+        OS - {os}
+    concurrent: true
+    logrotate:
+      daysToKeep: 30
+      numToKeep: 30
+    node: !include-jinja2: os2node.yaml.inc
+    scm:
+      - git:
+         url: https://github.com/kata-containers/{repo}
+         branches:
+          - '$sha1'
+    triggers:
+      - github-pull-request:
+          auth-id: "katacontainers bot connection"
+          github-hooks: true
+    builders:
+      - shell: |
+          #!/bin/bash
+          set -e
+          export ghprbPullId
+          export ghprbTargetBranch
+          export TEST_CRIO=true
+          export TEST_COMPATIBILITY=true
+
+          export GOPATH=${{WORKSPACE}}/go
+          export PATH=${{GOPATH}}/bin:/usr/local/go/bin:/usr/sbin:${{PATH}}
+          export GOROOT="/usr/local/go"
+
+          tests_repo="github.com/kata-containers/tests"
+          tests_repo_dir="${{GOPATH}}/src/${{tests_repo}}"
+          mkdir -p "${{tests_repo_dir}}"
+
+          git clone "https://${{tests_repo}}.git" "${{tests_repo_dir}}"
+          cd "${{tests_repo_dir}}"
+          git checkout "$ghprbTargetBranch"
+          .ci/jenkins_job_build.sh "github.com/kata-containers/{repo}"
+    publishers:
+      - post-tasks:
+          - matches:
+            - log-text: .*
+              operator: OR
+            script: |
+                #!/bin/bash
+
+                export GOPATH=$WORKSPACE/go
+                export GOROOT="/usr/local/go"
+                export PATH=${{GOPATH}}/bin:/usr/local/go/bin:/usr/sbin:/usr/local/bin:${{PATH}}
```

Comment:
- Could you use `$GOROOT/bin` here rather than hard-coding `/usr/local/go/bin`?
- Should`/usr/local/bin` be "higher up" than `/usr/sbin/`?

Reply:
To be honest I just copied the builder scripts from some of our current jobs, so I didn't pay attention to that kind of detail. I will fix it as we go.

Now that you asked, I noted that some jobs (maybe the majority?) simply clone the `tests` repo and call `tests/.ci/jenkins_job_build.sh`; others `curl` and execute `tests/.ci/ci_entry_point.sh`. So what's the ultimate entry point script for CI?